src/components/People/Welcome.js

```diff
@@ -4,12 +4,12 @@ import { Headline } from './Profile'
 function Welcome() {
   return (
     <Wrapper>
-      <Headline>Oh, ein Gast!</Headline>
+      <Headline>Programierer, Texter, Weltenretter!</Headline>
       <Text>
-        Hi, ich bin Falk aka Kritzelkopf und ich habe die Welt gerettet – vor
-        Lakritzel. Denn dieser übergeschnapste kleine Tropfen ist ein echter
-        Unruhestifter. Egal wo er auftaucht, versucht er die Menschen davon zu
-        überzeugen ihn zu Schlucken. <br /> <br />
+        und damit herzlich Willkommen! Ich bin Falk aka Kritzelkopf und ich habe
+        die Welt gerettet – vor Lakritzel. Denn dieser übergeschnapste kleine
+        Tropfen ist ein echter Unruhestifter. Egal wo er auftaucht, versucht er
+        die Menschen davon zu überzeugen ihn zu Schlucken. <br /> <br />
```

Comment:
"Egal wo er auftaucht, versucht er die Menschen davon zu überzeugen**,** ihn zu **s**chlucken." Not 100%, but I would write it like that.